source/jormungandr/jormungandr/street_network/tests/handimap_test.py

```diff
@@ -273,15 +273,15 @@ def get_language_handimap_func_language_valid_test():
 def get_language_parameter_handimap_func_language_invalid_test():
     instance = MagicMock()
     handimap = Handimap(instance=instance, service_url=fake_service_url)
-    request = {"_handimap_language": "toto"}
+    request = {"language": "toto"}
     language = handimap.get_language_parameter(request)
     assert language == "fr-FR"
 
 
 def get_language_parameter_handimap_func_language_invalid_test():
     instance = MagicMock()
     handimap = Handimap(instance=instance, service_url=fake_service_url)
-    request = {"_handimap_language": "english"}
+    request = {"language": "english"}
     language = handimap.get_language_parameter(request)
     assert language == "en-EN"
```

Comment:
if "english" is invalid (like "toto"), how can you have a different result?

Reply:
:+1: